components/StoryCards/HighlightedStoryCard.module.scss

```diff
@@ -84,6 +74,10 @@
     margin-bottom: 0;
 }
 
+.hugeTitle {
```

Comment:
Since we're using CSS modules I thought it would be better to use modifier classes, e.g. `&.huge` inside `.title` block.

That way in JSX you can just use `style.title` and `style.huge` and it will be automapped.

Reply:
Yeah, that makes sense! Thanks for the tip!